app_controller/pixi_frontend/src/ts/init_layout.ts

```diff
@@ -82,6 +81,12 @@ function connect_to_device() {
         characteristic.addEventListener('characteristicvaluechanged',
         send_next_bytes);
         console.log(characteristic);
+        let error = document.getElementById("error_msg");
+        error.innerHTML = "Connected successfully! You can now vroom vroom the robot";
```

Comment:
ASAP we should stop writing JS for our stuff and write React instead. This is fine for now though